packages/integration/src/bitbucket/config.ts

```diff
@@ -36,12 +36,12 @@ export type BitbucketIntegrationConfig = {
    * The base URL of the API of this provider, e.g. "https://api.bitbucket.org/2.0",
    * with no trailing slash.
    *
-   * May be omitted specifically for Bitbucket Cloud; then it will be deduced.
+   * May be omitted; then it will be deduced.
    *
    * The API will always be preferred if both its base URL and a token are
    * present.
    */
-  apiBaseUrl?: string;
+  apiBaseUrl: string;
```

Comment:
I don't think this need to be required right? Since the comment says `   * May be omitted; then it will be deduced.
`

Reply:
Maybe I should remove this comment entirely. It depends on the perspective. At the app-config level, users don't need to provide a value for it. But the reader creating the BitbucketIntegrationConfig instances will take care of creating the default values. Hence, BitbucketIntegrationConfig is guaranteed to always have apiBaseUrl set. No place anywhere else which is using BitbucketIntegrationConfig needs to manage a fallback on its own.

Reply:
I will make this clearer.